data/json/items/book/spiritual.json

```diff
@@ -117,6 +117,96 @@
     "volume": "1 L",
     "copy-from": "holybook_abstract"
   },
+  {
+    "id": "holybook_nag_hammadi",
+    "type": "BOOK",
+    "name": { "str": "Nag Hammadi Codex", "str_pl": "Nag Hammadi Codices" },
+    "description": "A translation of some of the gnostic texts and early biblical apocrypha recovered from the Nag Hammadi library in Egypt.  Obviously heretical, but the zombies aren't the type to discriminate.",
+    "weight": "281 g",
+    "volume": "1 L",
+    "copy-from": "holybook_abstract",
+    "variant_type": "generic",
+    "variants": [
+      {
+        "id": "holybook_nag_hammadi_01",
+        "name": { "str": "Nag Hammadi Codex I", "str_pl": "copies of the first Nag Hammadi Codex" },
+        "description": "It contains the Prayer of the Apostle Paul, the Apocryphon of James, the Gospel of Truth, the Treatise on the Resurrection, and the Tripartite Tractate.",
+        "append": true
+      },
+      {
+        "id": "holybook_nag_hammadi_02",
+        "name": { "str": "Nag Hammadi Codex II", "str_pl": "copies of the second Nag Hammadi Codex" },
+        "description": "It contains the Apocryphon of John (version 1), the Gospel of Thomas, the Gospel of Phillip, the Hypostasis of the Archons, On the Origin of the World, the Exegesis of the Soul, and the Book of Thomas the Contender.",
+        "append": true
+      },
+      {
+        "id": "holybook_nag_hammadi_03",
+        "name": { "str": "Nag Hammadi Codex III", "str_pl": "copies of the third Nag Hammadi Codex" },
+        "description": "It contains the Apocryphon of John (version 2), Holy Book of the Great Invisible Spirit, Eugnostos the Blessed, The Sophia of Jesus Christ, and the Dialogue of the Savior.",
+        "append": true
+      },
+      {
+        "id": "holybook_nag_hammadi_04",
+        "name": { "str": "Nag Hammadi Codex IV", "str_pl": "copies of the fourth Nag Hammadi Codex" },
+        "description": "It contains the Apocryphon of John (version 1) and Holy Book of the Great Invisible Spirit.",
```

Comment:
Is it intentional that both the texts you've mentioned in IV are also in II and III?

Reply:
Yes, the Nag Hammadi library has several copies of the same book, written by either different authors or copied by different individuals.

Reply:
That's cool I have no knowledge on the topic and were just checking you hadn't done it by mistake.
I don't see an issue with marking this as resolved as it's basically just noise. If the change leads to useful discussion say justifying the way you've done something over an alternative or just some useful stuff for future contributors I leave them unresolved so peeps can see them without digging but idk that there's any "etiquette".